docker_build/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# An integration test & dev container which builds and installs cuDF from main
+ARG CUDA_VERSION=10.2
+ARG CUDA_SHORT_VERSION=${CUDA_VERSION}
```

Comment:
IIRC this will only need the major.minor version here `11.2`

Reply:
Looks like to pull in the docker image we would have to use patch version too `11.2.2` works but `11.2` doesn't.